pkg/ipcache/ipcache.go

```diff
@@ -507,7 +507,7 @@ func (ipc *IPCache) deleteLocked(ip string, source source.Source) (namedPortsCha
 
 	cachedIdentity, found := ipc.ipToIdentityCache[ip]
 	if !found {
-		scopedLog.Debug("Attempt to remove non-existing IP from ipcache layer")
+		scopedLog.Warn("Attempt to remove non-existing IP from ipcache layer")
```

Comment:
FYI - caught this in an (unrelated) [draft PR](https://jenkins.cilium.io/job/Cilium-PR-K8s-1.26-kernel-net-next/138/testReport/junit/Suite-k8s-1/26/K8sDatapathConfig_AutoDirectNodeRoutes_Check_direct_connectivity_with_per_endpoint_routes/).